server/public/model/user.go

```diff
@@ -1039,6 +1040,59 @@ type UserReport struct {
 	UserPostStats
 }
 
+func (u UserReport) GetHeaders() []string {
+	return []string{
+		"Id",
+		"Username",
+		"Email",
+		"CreateAt",
+		"Name",
+		"Roles",
+		"LastLogin",
+		"LastStatusAt",
+		"LastPostDate",
+		"DaysActive",
+		"TotalPosts",
+	}
+}
+
+func (u *UserReport) ToReport() []string {
```

Comment:
The `ToReport` method is required for an object to be saved using the reporting module. It is responsible for morphing the data into a string format to be exported.